<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSongSelectionWidget</class>
 <widget class="QWidget" name="MultiSongSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="twSongsCorner" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnNew">
        <property name="text">
         <string>Přidat vše</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/resources.qrc">
          <normaloff>:/icons/16/Forward Button_16px.png</normaloff>:/icons/16/Forward Button_16px.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="twSongbookCorner" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnNew_2">
        <property name="text">
         <string> Vyčistit</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/resources.qrc">
          <normaloff>:/icons/16/Broom_16px.png</normaloff>:/icons/16/Broom_16px.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="twSongs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSongList">
      <attribute name="icon">
       <iconset resource="../../res/resources.qrc">
        <normaloff>:/icons/16/Musical Notes_16px.png</normaloff>:/icons/16/Musical Notes_16px.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Písně</string>
      </attribute>
      <layout class="QVBoxLayout" name="_2">
       <item>
        <widget class="SongListWidget" name="wgtSongList" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTabWidget" name="twSongbook">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSongList_3">
      <attribute name="icon">
       <iconset resource="../../res/resources.qrc">
        <normaloff>:/icons/16/Moleskine_16px.png</normaloff>:/icons/16/Moleskine_16px.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Zpěvník</string>
      </attribute>
      <layout class="QVBoxLayout" name="_4">
       <item>
        <widget class="QTreeWidget" name="twSelectedSongs">
         <column>
          <property name="text">
           <string>Název</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Autor</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SongListWidget</class>
   <extends>QWidget</extends>
   <header>wgt/songlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
